docs/global_flags.md

```diff
@@ -175,5 +175,16 @@ The tls cert flag `[--tls-cert]` specifies the public key for the client encoded
 ### tls-key
 The tls key flag `[--tls-key]` specifies the private key for the client encoded in PEM format
 
+### tls-max-version
+The tls max version flag `[--tls-max-version]` specifies the maximum supported TLS version supported by gNMIc when creating a secure gRPC connection
```

Comment:
`...specifies the maximum supported TLS version supported by gNMIc...`

maybe that wording is better?

```specifies the maximum supported TLS version offered by gNMIc```

Reply:
The TLS extension is called `supported_versions` that where that wording comes from.